src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
+ * TYPES
+ */
+
+type node = {
+  x: Number;
+  y: Number;
+  value: Number;
+  isExit: Boolean;
+};
+
+type edge = {
+  from: string;
+  to: string;
+  direction: string;
+};
+
+type Graph = {
+  nodes: {
+    [coordinates: string]: node;
+  };
+  edges: Array<edge>;
+};
+
+/**
+ * FUNCTIONS
+ */
+
+export function boardToGraph(board: Array<Array<Number>>): Graph | undefined {
+
+  if (board.length < 1) return undefined;
+
+  let graphFromBoard: Graph = newBlankGraph();
+
+
+  const verticalLines: Number = board.length;
+    const horizontalLines: Number = board[0].length;
```

Comment:
indentation problems. Yet prettier is already configured in your repo. You should turn on "format on save" in your IDE.

Reply:
Indentation problems ares solved in the [last commit](https://github.com/marmelab/kuba-integration/commit/52c48e332abaf5397cf6debd1512e2111062ba31)